cfn-resources/cluster/cmd/resource/mappings.go

```diff
@@ -292,15 +293,23 @@ func flattenReplicationSpecs(replicationSpecs []admin.ReplicationSpec) []Advance
 	return rSpecs
 }
 
-func flattenRegionsConfig(regionConfigs []admin.CloudRegionConfig) []AdvancedRegionConfig {
+func flattenRegionsConfig(regionConfigs *[]admin.CloudRegionConfig) []AdvancedRegionConfig {
+	if regionConfigs == nil {
+		return []AdvancedRegionConfig{}
+	}
+	regionConfigsVar := *regionConfigs
 	var regionsConfigs []AdvancedRegionConfig
```

Comment:
knit:  regionsConfigs:= make([]AdvancedRegionConfig, 0, len(regionConfigsVar)

also I think it's a bit confusing to use regionsConfigs & regionConfigsVar, I'll try to use a more differentiated name

Reply:
Thanks!
Renamed and fixed in b134b65